Telegram bot

```diff
@@ -0,0 +1,256 @@
+import numpy as np
+import math
+import telebot
+from telebot import types
+from peewee import *
+from datetime import datetime
+import threading, schedule
+import time as vremya
+
+
+bot = telebot.TeleBot('5245796353:AAHKpjAaha_JY9UDNLgSKjWSShLjlKaTf8I');
```

Comment:
There is a good instrument to work with secret tokens: https://github.com/theskumar/python-dotenv

And a constant string in the middle of the code seems not to be the best solution.

Moreover, you don't need `;` here